client/blocks/comment-detail/style.scss

```diff
@@ -0,0 +1,261 @@
+%comment-detail__ellipsis {
+	overflow: hidden;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+}
+
+%comment-detail__gray-link {
+	color: $gray;
+	cursor: pointer;
+	fill: $gray;
+	&:focus,
+	&:hover {
+		color: $blue-wordpress;
+		fill: $blue-wordpress;
+	}
+}
+
+.comment-detail.card {
+	font-size: 14px;
+	margin: 0 auto;
+	padding: 0;
+	transition: margin .15s linear;
+
+	&.is-expanded {
+		margin: 16px auto;
+	}
+	&:not( .is-expanded ):hover {
+		box-shadow: 0 0 0 1px $gray, 0 2px 4px lighten( $gray, 20% );
+		z-index: 1;
+	}
+
+	.gridicon {
+		margin-right: 4px;
+		vertical-align: middle;
+	}
+}
+
+.comment-detail__author-avatar {
+	flex-shrink: 0;
+	img {
+		border-radius: 50%;
+	}
+}
+.comment-detail__author-info {
+	@extend %comment-detail__ellipsis;
+	color: $gray;
+
+	strong {
+		color: $gray-text;
+		margin-right: 8px;
+	}
+}
+
+.comment-detail__header {
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: space-between;
+	padding: 12px 8px;
+
+	&.is-preview {
+		cursor: pointer;
+	}
+
+	.comment-detail.is-expanded & {
+		border-bottom: 1px solid lighten( $gray, 30% );
+	}
+
+	.comment-detail__author-avatar {
+		display: inline-block;
+		height: 24px;
+		padding: 0 8px;
+		vertical-align: middle;
+		width: 24px;
+	}
+
+	.comment-detail__comment-preview {
+		@extend %comment-detail__ellipsis;
+		padding: 0 8px;
+		width: 60%;
+	}
+
+	.comment-detail__actions {
+		color: $gray;
+		a {
+			@extend %comment-detail__gray-link;
+			padding: 8px;
+			white-space: nowrap;
+		}
+		span {
+			display: none;
+
+			@include breakpoint( ">960px" ) {
+				display: inline;
+			}
+		}
+	}
+}
+
+.comment-detail__post {
+	border-bottom: 1px solid lighten( $gray, 30% );
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: flex-start;
+	padding: 12px 16px;
+
+	.reader-avatar {
+		flex-shrink: 0;
+		margin-left: 0;
+	}
+
+	.comment-detail__post-info {
+		@extend %comment-detail__ellipsis;
+		color: $gray;
+		line-height: 32px;
+		padding-left: 4px;
+
+		span {
+			color: $gray-text;
+			margin-right: 8px;
+		}
+
+		a {
+			@extend %comment-detail__gray-link;
+		}
+	}
+}
+
+.comment-detail__comment {
+	border-bottom: 1px solid lighten( $gray, 30% );
+	padding-left: 16px;
+
+	.comment-detail__comment-content {
+		border-left: 3px solid lighten( $gray, 30% );
+	}
+
+	.comment-detail__comment-body {
+		padding: 0 16px;
+	}
+
+	.comment-detail__comment-reply {
+		padding: 16px;
+
+		a {
+			@extend %comment-detail__gray-link;
+		}
+	}
+}
+
+.comment-detail__author {
+	padding: 16px 0;
+
+	.comment-detail__author-info {
+		@extend %comment-detail__ellipsis;
+		> div {
```

Comment:
Can we add a classname to this div? We'd be able to avoid  the `>` and the deep nesting here